src/sentry/models/group.py

```diff
@@ -186,18 +173,20 @@ def from_event_id(self, project, event_id):
         return Group.objects.get(id=group_id)
 
     def filter_by_event_id(self, project_ids, event_id):
-        from sentry.models import EventMapping, Event
```

Comment:
Same issue, I think the check on EventMapping is still valid because it is the only way to fetch sampled events.

Reply:
I was under the impression that events are not sampled in Snuba (unlike Postgres) so we would not need to additionally check EventMapping.. is that not correct?

Reply:
Good quesiton. @tkaemming  ?

Reply:
This is correct, to my knowledge. 

I guess if we're going to be introducing a hard dependency on Snuba for events we should probably just go ahead and get rid of all vestiges of sampling (incl. `EventMapping`.)